packages/ember-metal/lib/mixin.js

```diff
@@ -39,6 +39,10 @@ import {
 } from 'ember-metal/events';
 import { isStream } from 'ember-metal/streams/utils';
 
+
+function K(){};
```

Comment:
this should just be called something other then `K`, ember already has `K` and it represents the K Combinator, which this does not.

Lets just call it Noop or something.